server/src/helpers/game/commands/gm/GMCreateGold.ts

```diff
@@ -0,0 +1,23 @@
+import { IMacroCommandArgs, IPlayer } from '../../../../interfaces';
+import { MacroCommand } from '../../../../models/macro';
+
+export class GMCreateGold extends MacroCommand {
+
+  aliases = ['@gold'];
+  isGMCommand = true;
+  canBeInstant = false;
+  canBeFast = false;
+
+  execute(player: IPlayer, args: IMacroCommandArgs) {
+    const gold = this.game.itemCreator.getSimpleItem('Gold Coin');
+    const value = +args.arrayArgs[0];
+    if (!value) {
+      this.sendMessage(player, `Enter an amount of gold.`);
+      return;
+    } else {
+      gold.mods.value = value;
+      this.game.worldManager.getMap(player.map).state.addItemToGround(player.x, player.y, gold);
```

Comment:
It should *probably* add it to your right hand if that's empty, first. If the right hand is empty, then drop it on the ground. You can see an example of this with the GM Item command.

Reply:
Thanks, done. It ended up having another else block similar to the old one, but this one doesn't have any returns so I assume we actually want this block